infrastructure/modules/Helm/main.tf

```diff
@@ -0,0 +1,34 @@
+module load-balancer-controller {
+  source  = "terraform-module/release/helm"
+  version = "~> 2.8.0"
+
+  namespace  = "vegait-load-balancer"
```

Comment:
add parameter to create namespace if it does not exist.
At least that is needed for the resource, I don't know what module does by default if it doesnt exist.

Reply:
I will add it, module use false as default value.